src/app/components/Header.tsx

```diff
@@ -27,6 +27,7 @@ export default function Header() {
 				{ name: 'Nohooks', route: 'https://nohooks.io', type: 'external' }
 			]
 		},
+		{ name: 'About Us', route: '/aboutus', type: 'route' },
```

Comment:
@Oluwadaminiola This shouldn't be on the nav bar. It should be in the footer under `Company`.